### PR TITLE
Version 2.0.0

### PR DESCRIPTION
**Pull request by mivade**
Need to get the following tests passing with rhino access:

- [x] `test_classifier.py::TestClassifier::test_classifier`
- [x] `test_talreader.py::test_old_behavior_warning`
- [x] `test_timeseries.py::test_hdf`

### CODE REVIEW

**Comment by mivade** [comment]
As luck would have it, PR 200 is for PTSA 2.0.0!

**Comment by mivade** [comment]
@ctw once you approve I'll update the release date and ensure the conda package builds before merging and releasing.

File: docs/conf.py [hunk 1]
```diff
@@ -61,7 +62,7 @@
 
 # General information about the project.
 project = u'PTSA'
-copyright = u'2017'
+copyright = str(datetime.now().year)
```
Comment (ctw) [inline docs/conf.py]: We should probably change this to 2018 and update whenever we do edits.

Comment (mivade) [reply docs/conf.py]: That's exactly what this does...

**Comment by mivade** [comment]
The warning check fails on legacy Python for some reason, but I don't really care about that.

**Pull request merged**